osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -189,6 +183,12 @@ public override void OnResuming(IScreen last)
 
             //we may have consumed our preloaded instance, so let's make another.
             preloadSongSelect();
+
+            if (Beatmap.Value.Track != null && !musicController.UserRequestedPause)
+            {
+                Beatmap.Value.Track.Tempo.Value = 1;
```

Comment:
This could probably use a comment why it's there because the only way to get into `MainMenu` and keep adjustments on the track is leaving the `Editor` with 50%/75% speed.
Although it was intended to start the track here I'm not sure if the `Editor` should be at least responsible for removing any adjustments that were made there when leaving, like `SongSelect` does. Feels inconsistent otherwise.

Reply:
I agree. I'll make the changes